apps/sherbet_service/lib/sherbet.service/contact/communication/method/mobile.ex

```diff
@@ -4,6 +4,7 @@ defmodule Sherbet.Service.Contact.Communication.Method.Mobile do
     """
 
     @behaviour Sherbet.Service.Contact.Communication.Method
+    @key Application.get_env(:sherbet_service, :mobile_key_length, 8)
```

Comment:
Same as the email, it would be better to move this just above the `generate_key` function. Also the default value should be changed to `5`, as that was the original length.